src/main.rs

```diff
@@ -1,3 +1,47 @@
-fn main() {
-    println!("Hello, world!");
+use clap::Parser;
+use starknet_adapter::args::Cli;
+use starknet_adapter::cairo1::run_cairo1;
+use starknet_adapter::prover::run_stone_prover;
+use starknet_adapter::utils::{cleanup_tmp_files, set_env_vars};
+use starknet_adapter::verifier::run_stone_verifier;
+use tempfile::Builder;
+
+const CONFIG: &[u8] = include_bytes!("../configs/env.json");
```

Comment:
You can `include_str` as well: https://doc.rust-lang.org/std/macro.include_str.html

Reply:
Done